src/test/regress/sql/reassign_owned.sql

```diff
@@ -0,0 +1,140 @@
+CREATE ROLE distributed_source_role1;
+create ROLE "distributed_source_role-\!";
+
+CREATE ROLE distributed_target_role1;
```

Comment:
can we name this as such in order to test whether we properly quote target role?

```sql
"distributed_target_role1-\!""
```

Reply:
Applied suggestion